index.html

```diff
@@ -1,124 +1,116 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <header>
-      <div class="My-Logo">
-        <a href="#"><h1>My Logo</h1></a>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header>
+    <div class="My-Logo">
+      <a href="#">
+        <h1>My Logo</h1>
+      </a>
+    </div>
+    <nav class="header-Line-parent">
+      <img src="Union.svg" alt="menu-icon" />
+    </nav>
+  </header>
+
+  <main>
+    <!-- home section 1 -->
+    <section class="section-1">
+      <img class="background-img" src="header-shapes mobile@2x.svg" alt="" />
+      <div class="heading-1">
+        <h1 class="First-heading">
+          I'm Hasnat <br />
+          Glad to see <br />
+          you!
+        </h1>
+        <p class="First-heading-description">
+          I'm a software developer! I can help you build a prduct,feature or
+          website Look through some of my work and experience! if you like
+          what you see and have a project you need coded, don't hesitate to
+          contact me.
+        </p>
```

Comment:
- [ ] You did well implementing most of the requirements however, the header is not displayed and the cards are not displayed properly as you can see in the caption. Kindly fix the issue.
- [ ] The button on hover still not matching the Figma design 

![review](https://user-images.githubusercontent.com/53347966/219900299-bc051acd-94bd-4024-99ec-89b671dadab2.JPG)

Reply:
hi.. can you tell me Please from where I found hower color which matching the figma design